client/src/components/messages/ChatSearchInput.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import { BiSearch } from "react-icons/bi";
+import { MdFilterList } from "react-icons/md";
+
+const ChatSearchInput = ({ text, placeholder }) => {
+  const [filtered, setFiltered] = useState(false);
+  const handleFilter = () => {
+    setFiltered((prev) => !prev);
+  };
+  return (
+    <div className=" flex items-center w-full">
+      <div className="relative flex items-center rounded-lg  bg-slate-100 w-full ">
+        <input
+          type={text}
+          placeholder={placeholder}
+          className="border-0 pl-6 pr-2 py-2 focus:outline-none bg-transparent w-full "
+        />
+        <BiSearch className="absolute left-1" />
+      </div>
+
+      {filtered ? (
+        // Filter for unread messages
+        <div
+          className="bg-grotto-100 text-white ml-2 text-lg flex items-center justify-center w-6 h-6 rounded-full cursor-pointer"
+          onClick={handleFilter}
+        >
+          <MdFilterList />
+        </div>
+      ) : (
+        <div className="w-6 h-6 ml-2 flex items-center justify-center text-lg cursor-pointer">
+          <MdFilterList onClick={handleFilter} />
```

Comment:
If we pass this the prop color="#265B8B" it'll take the primary color we've been using and make it more consistent with the overall color palette

Reply:
Maybe you can do this so I can checkout how it looks

Reply:
It'sjust pasting `color="#265B8B"` into the component.